doc/contribute/committer_expectations.rst

```diff
@@ -0,0 +1,248 @@
+  reviewer comments. This provides reviewers a clear history of the changes made
+  to the PR, along with the rationale for changes. Prior to the final approval
+  and merge of the PR, you must perform a squash merge to cleanup the commit
+  history. (TBD - This needs a PoC to demonstrate this workflow).
+
+PR Escalation
+=============
+
+The Zephyr community is a diverse group of individuals, with different levels of
+commitment and priorities. As such, reviewer and maintainers may not get to your
+PR right away.
+
+The Zephyr Project recommends the following approaches to escalate reviews for
+your PR.
+
+- After 2 days of inactivity, post a message on the `#pr-help`_ channel on
+  Discord linking to your PR.
+
+- After 1 week of inactivity, add the `dev-review`_ label to your PR. This adds
+  your PR to the agenda for the next `Zephyr Dev Meeting`_.
+
+- If reviewers on your PR propose mutually exclusive or otherwise incompatible
+  changes, add your PR to the `Architecture Project`_. This marks your PR for
+  triage by the `Architecture Working Group`_.  [#arch-group]_
+
+- In rare instances, the `Zephyr Dev Meeting`_ and `Architecture Working Group`_
+  may fail to reach consensus on the direction/design of a PR. In these cases,
+  you may add the `tsc`_ label to your PR to escalate to the Zephyr Technical
+  Steering Committee.
+
+- Members of the :ref:`release-engineering-team` have the authority to dismiss
+  stale reviews if they determine the PR addressed the requested changes or
+  the requested changes no longer apply.
+
+
+.. _#pr-help: https://discord.com/channels/720317445772017664/997527108844798012
+
+.. _dev-review: https://github.com/zephyrproject-rtos/zephyr/labels/dev-review
+
+.. _Zephyr Dev Meeting: https://github.com/zephyrproject-rtos/zephyr/wiki/Zephyr-Committee-and-Working-Group-Meetings#zephyr-dev-meeting
+
+.. _Architecture Project: https://github.com/zephyrproject-rtos/zephyr/projects/18
+
+.. _Architecture Working Group: https://github.com/zephyrproject-rtos/zephyr/wiki/Architecture-Working-Group
+
+.. _tsc: https://github.com/zephyrproject-rtos/zephyr/labels/tsc
+
+Reviewer Expectations
+#####################
+
+- Be respectful when commenting on PRs. Refer to the Zephyr `Code of Conduct`_
+  for more details.
+
+- The Zephyr Project recognizes that reviewers and maintainers have limited
+  bandwidth. Prioritize review requests in the following order:
+
+    #. PRs related to items in the `Zephyr Release Plan`_.
+    #. PRs you previously reviewed and requested blocking changes.
+    #. PRs assigned to you as the area maintainer.
+    #. All other PRs.
+
+- Try to provide feedback on the entire PR in one shot. This provides the author
+  an opportunity to address all comments in the next PR update.
+
+- Partial reviews are permitted, but the reviewer must add a comment indicating
+  what portion of the PR they reviewed. Examples of useful partial reviews
+  include:
```

Comment:
Not sure what is the best place but I have explained to very many people new to Github that they too can use the "Start Review" button on their _own_ Pull Requests to reduce notification spam by one order of magnitude - and also get a chance to proof-read their comments.

Reply:
Added and commented above, resolving here.